src/applications/vaos/containers/ReasonForAppointmentPage.jsx

```diff
@@ -44,9 +44,11 @@ const uiSchema = {
 };
 
 const pageKey = 'reasonForAppointment';
+const pageTitle = 'Reason for appointment';
 
 export class ReasonForAppointmentPage extends React.Component {
   componentDidMount() {
+    document.title = `${pageTitle}  | Veterans Affairs`;
```

Comment:
It looks like there's sometimes one space and sometimes two spaces in between the title and `|`

Reply:
@jbalboni, good eye to spot the extra space!